scripts/ci.ps1

```diff
@@ -6,5 +6,4 @@ Import-Module -WarningAction Ignore -Name "$PSScriptRoot\utils.psm1"
 
 Invoke-Script -File "$PSScriptRoot\test.ps1"
 Invoke-Script -File "$PSScriptRoot\build.ps1"
-
-Invoke-Expression -Command "$PSScriptRoot\package.ps1"
+Invoke-Script -File "$PSScriptRoot\package.ps1"
```

Comment:
nit change, can be reverted. Don't think it has any consequence but changed this to be consistent.